bin/lib/common_service.sh

```diff
@@ -90,16 +88,15 @@ service_start() {
     done
 
     # (1) do not start if service already running
-    ## pgrepf  "\b${F_BINDIR}/${F_SERVICE_CMD} .* --config ${F_BASENAME}[0-9].toml\b"
     PLIST=$(pgrepf  "${F_BINDIR}/${F_SERVICE_CMD} .* --config ${F_BASENAME}[0-9].toml\b")
     if [ -n "$PLIST" ] ; then
         echo existing ${F_SERVICE_NANME} services detected, please shutdown first
         exit 1
     fi
 
     # (2) start services asynchronously
-    for index in `seq 1 $F_COUNT` ; do
-        IDENTITY="${F_BASENAME}$index"
+    ILIST=$(basename --suffix=.toml ${F_CONFDIR}/${F_BASENAME}[0-9].toml)
```

Comment:
We're not assuming that we'll have more than 10 services running at once? I'm ok with that, but might want to document that assumption somewhere.

Reply:
Not sure how i feel about the 10 limit. I'll adjust.

Reply:
Done